install/update_91_92.php

```diff
@@ -1870,6 +1870,22 @@ function update91to92() {
    //wrong type
    $DB->queryOrDie("ALTER TABLE `glpi_users` CHANGE `keep_devices_when_purging_item` `keep_devices_when_purging_item` TINYINT(1) NULL DEFAULT NULL");
 
+   //missing index
+   $migration->addKey('glpi_networknames', 'is_deleted');
+   $migration->addKey('glpi_networknames', 'is_dynamic');
+   $migration->addKey('glpi_projects', 'is_template');
+   $migration->addKey('glpi_projecttasks', 'is_template');
+   $migration->dropKey('glpi_savedsearches_users', 'bookmarks_id');
+   $migration->addKey('glpi_savedsearches_users', 'savedsearches_id');
+   //this one was not on the correct field
+   $migration->dropKey('glpi_softwarelicenses', 'is_deleted');
+   $migration->migrationOneTable('glpi_softwarelicenses');
+   $migration->addKey('glpi_softwarelicenses', 'is_deleted');
```

Comment:
Because the original created index was not on the right field :/

We had sthing like `KEY is_deleted (is_visible)`